x-sounds-mac/Domain/XSound/XSoundsEndpointService.swift

```diff
@@ -3,13 +3,15 @@ import Foundation
 import SwiftUI
 
 struct XSoundsEndpointService {
-    @AppStorage(AppStorageKeys.FMinstancePath.rawValue) static var instancePath = ""
-    
-    struct URLCreatingError: Error { }
-    
+    @AppStorage(AppStorageKeys.fmInstancePath.rawValue) static var instancePath = ""
+
     static func getSounds(webService: WebServiceProtocol) -> AnyPublisher<[XSound], Error> {
-        let urlString = instancePath + "x-sounds"
-        var request = URLRequest(url: URL(string: urlString)!)
+        guard let url = URL(string: instancePath) else {
+            return Fail<[XSound], Error>
+                .init(error: WebService.Error.invalidURL)
+                .eraseToAnyPublisher()
+        }
+        var request = URLRequest(url: url.appendingPathComponent("x-sounds", isDirectory: true))
```

Comment:
Shouldn't `isDirectory` be false?
```suggestion
        var request = URLRequest(url: url.appendingPathComponent("x-sounds", isDirectory: false))
```

It should do `GET /x-sounds`, not `GET /x-sounds/`